Docs/Actions/ReplaceTokensInText.md

```diff
@@ -142,7 +142,7 @@ $(TIMER_FUNCTION_BODY)
 
 ```python
 @app.function_name(name="$(FUNCTION_NAME_INPUT)")
-@app.schedule(schedule="$(SCHEDULE_INPUT)", arg_name="myTimer", run_on_startup=True,
+@app.timer_trigger(schedule="$(SCHEDULE_INPUT)", arg_name="myTimer", run_on_startup=False,
```

Comment:
Curious to understand the context behind changing `run_on_startup` to `False`?

Reply:
run_on_startup should always be false by default. Its default in V1 so one reason is for parity. But main reasons why this should always be false by default is for production, we don't want customers to accidentally create too many timers that run on startup. Updated the description with more details.